schema/organization/Person.md

```diff
@@ -0,0 +1,130 @@
+# Person
+
+The `Person` type allows you to provide details about a person such as their given and family names, any honourifix prefix or suffix, and contact details such as an email address. This type of often used to describe the `authors` of an [`Article`](/Article), or other [`CreativeWork`](/CreativeWork).
+
+## Examples
+
+These examples, based on Marie Curie, illustrate alternative ways for specifying a `Person`. [Marie Curie](https://en.wikipedia.org/wiki/Marie_Curie) was the first woman to win a Nobel Prize, the first person and only woman to win twice, and the only person to win a Nobel Prize in two different sciences.
+
+<!--
+These examples will eventually be wrapped in React components
+to illustrate how the input is converted into Stencila JSON
+See https://github.com/stencila/schema/issues/45
+-->
+
+Dr Curie can be represented in canonical Stencila JSON by:
+
+```json
+{
+  "type": "Person",
+  "honorificPrefix": "Dr",
+  "givenNames": ["Marie", "Skłodowska"],
+  "familyNames": ["Curie"],
+  "honorificSuffix": "PhD"
+}
+```
+
+YAML provides a more readable format for providing details about a person in places like Markdown front-matter. In the following example, we take advantage of [property aliases](/docs/property-aliases) to use the shorter `prefix` and `suffix` property names, and the US convention of `fistNames` and `lastName` (instead of `givenNames` and `familyNames`). We also use [property parsing](/docs/property-parsing) to be be able to write `fistNames` as a space separated values.
+
+```markdown
+---
+title: Recherches sur les substances radioactives
+datePublished: 1904
+authors:
+  - prefix: Dr
+    firstNames: Marie Skłodowska
+    lastName: Curie
+    suffix: PhD
+---
+
+Le présent travail a pour but d'exposer les recherches que je poursuis depuis plus de 4 ans sur les substances radioactives. J'ai commencé ces recherches par une étude du rayonnement uranique cjui a été découvert par M. Becquerel. Les résultats auxquels ...
+```
+
+We can shorten this further by specifying Dr Curie's details as a string:
+
+```yaml
+type: Article
+title: Recherches sur les substances radioactives
+datePublished: 1904
+authors:
+  - Dr Marie Skłodowska Curie PhD
+```
+
+If there had been email and web pages in the 1900s then we could also add those for her and her collegues:
+
+```yaml
+type: Article
+title: The Radioactive Constants as of 1930 Report of the International Radium-Standards Commission
+year: 1931
+url: https://link.aps.org/doi/10.1103/RevModPhys.3.427
+authors:
+  - Marie Curie <marie@mariecurie.org.uk> (https://www.mariecurie.org.uk/)
+  - Debierne, A.
+  - Eve, A. S.
+  - Geiger, H.
+  - Hahn, O.
+  - Lind, S. C.
+  - Meyer, St.
+  - Rutherford, Ernest (https://en.wikipedia.org/wiki/Ernest_Rutherford)
+  - Schweidler, E.
+```
+
+## Related
+
```

Comment:
We should reference the relevant ISA schema here as well https://isa-specs.readthedocs.io/en/latest/isajson.html#person-schema-json

Reply:
Added section for ISA person